File: daily question/412. Fizz Buzz/java/Fizz Buzz.java
// 日期：2021.10.13
class Solution {
    public List<String> fizzBuzz(int n) {
        List<String> data = new ArrayList();
        for(int i = 1; i <= n ;i++) {
            boolean flag_3 = i%3 == 0;
            boolean flag_5 = i%5 == 0;

            if(!flag_3 && !flag_5) {
                data.add(String.valueOf(i));
            }
            if(flag_3 && flag_5) {
                data.add("FizzBuzz");
            }
            if(flag_3 && !flag_5) {
                data.add("Fizz");
            }
            if(!flag_3 && flag_5) {
                data.add("Buzz");
            }
        }

        return data;
    }
}

File: daily question/412. Fizz Buzz/python/Fizz Buzz.py
# @lc code=start
class Solution:
    def fizzBuzz(self, n: int) -> List[str]:
        result = []
        for i in range(1,n):

            if i % 3 == 0 and i % 5 ==0:
                result.append('FizzBuzz')
            elif i % 3 == 0:
                result.append('Fizz')
            elif i % 5 == 0:
                result.append('Buzz')
            else:
                result.append(i)
        return result

